.github/workflows/publish.yml

```diff
@@ -0,0 +1,58 @@
+name: Build and push package
+
+on:
+  workflow_dispatch:
+    inputs:
+      nr_launcher_release_name:
+        description: 'flowforge-nr-launcher package version'
+        required: false
+        default: 'nightly'
+      nr_launcher_ref:
+        description: 'flowforge-nr-launcher package ref'
+        required: false
+  schedule:
+    - cron: '0 23 * * *'
+  pull_request:
+    branches: 
+      - main
+  push:
+    branches:
+      - main
+
+
+jobs:
+  build:
+    uses: 'flowforge/github-actions-workflows/.github/workflows/build_node_package.yml@main'
+
+  publish:
+    needs: build
+    permissions:
+      contents: read
+      packages: write
+    uses: 'flowforge/github-actions-workflows/.github/workflows/publish_node_package.yml@main'
+    with:
+      package_name: flowforge-driver-localfs
+      publish_package: true
+      package_dependencies: |
+        @flowforge/nr-launcher
```

Comment:
I assume this means, once this package has been built, buld `@flowforge/nr-launcher` as it depends on this package?

Reply:
At this moment this parameter is used in the task which is responsible for setting proper dependencies versions in _package.js_ file.